core-services/prow/02_config/konflux-ci/build-definitions/_pluginconfig.yaml

```diff
@@ -26,4 +26,34 @@ external_plugins:
 plugins:
   konflux-ci/build-definitions:
     plugins:
+    - assign
+    - blunderbuss
+    - cat
+    - dog
+    - heart
```

Comment:
Do we need zoo here?

Reply:
these are some interesting plugins part of list of supported [plugins](https://prow.ci.openshift.org/plugins) .. just to make the interaction funny... since I took list of enabled plugins from infra-deployment repo and  it was added there.. kept them as it is..

Reply:
enabled only relevant plugins needed for review assignment now